### PR TITLE
Intro to Regular Expressions and Linux Tools -- Katelyn Spilman

### PR DESCRIPTION
**Pull request by katelynspilman**
I wrote this blog post to provide beginning bioinformaticians with a high level introduction to regular expressions and the Linux command line tools grep, sed and awk.  Please let me know if you have any issues, comments or suggestions. Thank you.

### CODE REVIEW

File: _posts/2020-11-07-Intro-to-Regular-Expressions-and-Linux-Tools.md [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+apple
+dog
+first
+Eat
+blue
+catnap
+Green
+```
+If we want to find all instances within `sample.txt`
+that exactly match the word "dog", we can use the following `grep` command:
```
Comment (jsandrad) [inline _posts/2020-11-07-Intro-to-Regular-Expressions-and-Linux-Tools.md]: you used grep cat, not grep dog.
one way to make this more clear, have multiple dogs in the sample.txt so it pulls multiple lines

File: _posts/2020-11-07-Intro-to-Regular-Expressions-and-Linux-Tools.md [hunk 2]
```diff
@@ -0,0 +1,231 @@
+---
+layout: post
+title:  "Intro to Regular Expressions & Linux Tools: grep, sed, and awk"
+date:   2020-11-07 12:54:04 -0700
+categories: regular expressions grep sed awk
+---
+Written by: Katelyn Spilman
```
Comment (jsandrad) [inline _posts/2020-11-07-Intro-to-Regular-Expressions-and-Linux-Tools.md]: use the "#" to add title headers and "###" sub-header as your name

**Review by jsandrad — CHANGES_REQUESTED** [review]
pretty good! added some comment, i like it when you explain every bit of code in the expressions because it helps me learn them better

File: _posts/2020-11-07-Intro-to-Regular-Expressions-and-Linux-Tools.md (hunk 1 above)
Comment (katelynspilman) [reply _posts/2020-11-07-Intro-to-Regular-Expressions-and-Linux-Tools.md]: Thanks for catching that! I meant to put cat. Fixed by 0d2e4bf91ae4b975327223b7231fbe1e1f08f293

File: _posts/2020-11-07-Intro-to-Regular-Expressions-and-Linux-Tools.md (hunk 2 above)
Comment (katelynspilman) [reply _posts/2020-11-07-Intro-to-Regular-Expressions-and-Linux-Tools.md]: Fixed by 3af724dd43562936b0e86ecf8247464b48174578

**Comment by jsandrad** [comment]
looks great. need more resources like this on the internet!

**Review by mhaliz — APPROVED** [review]
Great, detailed explanation on different regex tools and their applications!

**Pull request merged**